certz/certz.proto

```diff
@@ -452,6 +479,15 @@ message CertificateRevocationListBundle {
   repeated CertificateRevocationList certificate_revocation_lists = 1;
 }
 
+// An authentication policy.
+// This policy defines which Certificate Authority can sign which certificate.
+message AuthenticationPolicy {
+  // Actual authentication policy.
+  oneof policy {
+    google.protobuf.Any serialized = 1;
```

Comment:
Is this a well-known authentication policy which is accepted via some API by grpc stack or TLS stack (openssl for example)? In other words, is there a way to configure grpc stack to allow or reject the clients based on this policy? Because the grpc stack handles TLS handshake with the clients.

Reply:
As I have written above the protobuf definition for the description of the policy is being worked on as well as code that will be part of the gRPC framework/generated code that will handle it.

Reply:
Ok, this policy needs to be fed to gRPC stack so that it understands the policy and acts accordingly. Makes sense. Once the policy is published publicly and integrated with gRPC framework, can you share the reference of it here? Thanks!

Reply:
Will do.